src/cache-restore.ts

```diff
@@ -0,0 +1,50 @@
+import {readdir} from 'node:fs/promises';
+import {join} from 'node:path';
+import * as cache from '@actions/cache';
+import * as core from '@actions/core';
+import * as glob from '@actions/glob';
+
+import {getNuGetFolderPath} from './cache-utils';
+import {lockFilePatterns, State, Outputs} from './constants';
+
+export const restoreCache = async (cacheDependencyPath?: string) => {
+  const lockFilePath = cacheDependencyPath || (await findLockFile());
+  const fileHash = await glob.hashFiles(lockFilePath);
+  if (!fileHash) {
+    throw new Error(
+      'Some specified paths were not resolved, unable to cache dependencies.'
+    );
+  }
+
+  const platform = process.env.RUNNER_OS;
+  const primaryKey = `dotnet-cache-${platform}-${fileHash}`;
+  core.debug(`primary key is ${primaryKey}`);
+
+  core.saveState(State.CachePrimaryKey, primaryKey);
+
+  const {'global-packages': cachePath} = await getNuGetFolderPath();
+  const cacheKey = await cache.restoreCache([cachePath], primaryKey);
+  core.setOutput(Outputs.CacheHit, Boolean(cacheKey));
+
+  if (!cacheKey) {
+    core.info('dotnet cache is not found');
```

Comment:
```suggestion
    core.info('Dotnet cache is not found');
```
**Nit:** Just a small change to make logs format consistent across the whole action.